src/components/education/EducationForm.jsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from "react"
+
+const EducationForm = () => {
+    const [formData, setFormData] = useState ({
+        course: '',
+        school: '',
+        start_date: '',
+        end_date: '',
+        grade: '',
+        logo: ''
+    })
+
+    const handleChange = (field) => (e) => {
+        e.preventDefault();
+
+        if (field === 'grade') {
+            setFormData({
+                ...formData,
+                [field]: `${e.currentTarget.value}%`
+            })
+        } else {
+            setFormData({
+                ...formData,
+                [field]: e.currentTarget.value
+            })
+        }
+    }
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+        fetch('resume/education', {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify(formData)
+        })
+        .then(response => response.json())
+        .then(data => {
+            console.log('Success:', data)
```

Comment:
We could show an alertbox here since this success is relevant for the user

Reply:
Added alert box for successful and unsuccessful submission